compute/src/main/java/org/jclouds/compute/domain/internal/TemplateBuilderImpl.java

```diff
@@ -131,6 +129,7 @@
    @VisibleForTesting
    protected Boolean forceCacheReload;
 
+   @VisibleForTesting
```

Comment:
It is not just visible for testing; it is visible to allow inheritance, so remove this annotation.